includes/class-wc-calypso-bridge-ecommerce-admin-menu.php

```diff
@@ -176,7 +176,7 @@ public static function menu_order( $menu_order ) {
 	 * Adds My Home menu.
 	 */
 	public function add_my_home_menu() {
-		$this->update_menu( 'index.php', 'admin.php?page=wc-admin', __( 'My Home', 'jetpack' ), 'edit_posts', 'dashicons-admin-home' );
+		$this->update_menu( 'index.php', 'admin.php?page=wc-admin', __( 'My Home', 'wc-calypso-bridge' ), 'edit_posts', 'dashicons-admin-home' );
```

Comment:
@daledupreez Thank you for taking the time to provide feedback. Instead of immediately committing these changes, I'll wait for input from other teams. 

I suspect that I'll need to add `phpcs:ignore WordPress.WP.I18n` comments based on their feedback